gemrb/core/CMakeLists.txt

```diff
@@ -174,7 +175,7 @@ IF (APPLE)
 		LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/gemrb/${PROJECT_NAME}.app/Contents/Frameworks
 	)
 
-	IF (BUILD_TESTING)
+	IF (USE_TESTS)
```

Comment:
One more at gemrb/CMakeLists.txt:173

Reply:
bah, yeah, trying to get tests to work without luck